controllers/comments.controller.js

```diff
@@ -14,3 +18,22 @@ exports.getCommentsByArticle = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postCommentToArticle = (req, res, next) => {
+  const {
+    params: { article_id },
+    body: { username, body },
+  } = req;
+
+  const commentPromises = [
+    selectArticleById(article_id),
```

Comment:
- author in the comments table is a foreign key which relates to username in the users table. A psql error will be thrown if this is violated. It is not necessary to separately check if the user exists.
- article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown.